lambdas/README.md

```diff
@@ -1,12 +1,64 @@
 # Lambdas
 
-This lambda is to update the view dynamo table from the main one.
+These lambdas relate to automated tasks, run on a daily or weekly basis, which copy promotions data stored in the DynamoDB tables over to SalesForce and BigQuery, as well as backing up the DB tables themselves.
```

Comment:
are any of them weekly?

Reply:
I thought the backup was weekly? I may have got a bit confused during your testing yesterday.

Reply:
Copy updated